index.html

```diff
@@ -1,11 +1,41 @@
-Ki Vera Luna Pardillo
+<!DOCTYPE html>
+<html>
+<head>
+<style> 
 
-"About"
+body {background-color: powderblue;}
+h1 {color: black}
 
-My name is Ki Vera Luna. I am from the Philippines. My major is Computer Information System.
+</style>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="keywords" content="HTML, CSS, JavaScript">
+    <meta name="author" content="Ki Vera Luna Webpage">
 
-"Experience"
--Worked in GTC
+</head>
 
-"Connect"
-https://www.linkedin.com/in/kipardillo/
\ No newline at end of file
+<body>
+
+<h1>Ki Vera Luna Pardillo</h1>
+
+<section>
+<h1>About</h1>
+<p>My name is Ki Vera Luna. I am from the Philippines. My major is Computer Information System. </p>
+</section>
+
+<section>
+<h1>Experience </h1>
+<ul>
+    <li>Gateway Technology Center Receptionist</li>
+    <li>Geodetic Survey Summer Intern</li>
+</ul>
+</section>
+
+<section>
+<h1>Connect </h1>
+<a href= https://www.linkedin.com/in/kipardillo/> linkedin </a>
```

Comment:
I think this _technically_ works since there's no spaces in the url or anything, but correct syntax is to always use quotation marks (`"`) for the HTML attributes

Eg:

`<a href="https://www.linkedin.com/in/kipardillo/">linkedin</a>`